packages/ragbits-document-search/src/ragbits/document_search/_main.py

```diff
@@ -141,8 +141,26 @@ async def ingest(
         elements = await self.processing_strategy.process_documents(
             documents, self.document_processor_router, document_processor
         )
+        await self.remove_entries_with_same_id(elements)
         await self.insert_elements(elements)
 
+    async def remove_entries_with_same_id(self, elements: list[Element]) -> None:
```

Comment:
maybe `remove_entries_with_same_sources`? Looking at the current name, I would expect this to be based on ids of the elements, not on the ids of sources

Reply:
Oh, great idea!